std_msgs/Header header
geometry_msgs/Pose pose
string category
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status
geometry_msgs/Pose pre_pick_pose
geometry_msgs/Pose pick_pose
geometry_msgs/Pose middle_pose
geometry_msgs/Pose place_pose